spec/index.bs

```diff
@@ -313,6 +313,10 @@ dictionary GPUBindGroupLayoutBinding {
     // are more efficient to bind at run-time.
     GPUTextureViewDimension textureDimension;
     bool multisample = false;
+    // For "sampler" binding type, if the actual sampler object is provided here,
+    // it becomes a part of the layout: an "immutable" sampler object, in a sense that
+    // no bind group binding call can affect it.
+    GPUSampler immutableSampler;
```

Comment:
Would it make sense for this to be a GPUSamplerDescriptor instead? It doesn't match what Vulkan does but is closer to what D3D12 and Metal do where the sampler descriptor is embedded in the shading language source.

Reply:
You mean, having a flag there saying that a sampler is immutable? That's not going to be known at pipeline creation time, so I don't see how this would help. Please clarify.

Reply:
I think Kangz means providing the creation descriptor instead of the platform object to the layout. I'm not sure I like that as much, but I see the appeal of it.

Reply:
Oh I see. Thanks for clarification! I think that would make sense to do.

Reply:
this change is now in